File: samples/Go/pick-random-element.go
package main

import (
    "fmt"
    "math/rand"
    "time"
)

var list = []string{"bleen", "fuligin", "garrow", "grue", "hooloovoo"}

func main() {
    rand.Seed(time.Now().UnixNano())
    fmt.Println(list[rand.Intn(len(list))])
}

File: samples/C/pick-random-element.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(){
  char array[] = { 'a', 'b', 'c' };

  srand( time( 0 ) );

  printf( "%c\n", array[ rand() % 3 ] );

  return 0;
}
